packages/core/demo/index.ts

```diff
@@ -231,6 +231,11 @@ chartTypes.forEach(type => {
 				);
 
 				const chartObject = classyCharts[type.id];
+				chartObject.events.addEventListener("onClick", e=> {
+					console.log("Im clicked");
+					console.log(e.detail);
+				})
+
```

Comment:
1. We should use more descriptive text here, e.g. `Bar slice clicked`.
2. `console.log("Im clicked", e.detail);` would be simpler
3. The name `onClick` is misleading since it could mean catching a click anywhere on the pie chart, should use `bar-onClick`